docs/workflow/debugging/libraries/mobile-instructions.md

```diff
@@ -0,0 +1,6 @@
+Debugging Libraries on Mobile
+==========================
+
+## Debugging System.Private.CoreLib on Android
+
+`System.Console.Write`/`System.Console.WriteLine` cannot be used in `System.Private.CoreLib`. Instead, use `Internal.Console.Write` to log. After running a test or program that utilizes the relevant `System.Private.CoreLib` code, the logs can be found through the generated adb log or viewed directly through adb logcat.
```

Comment:
This is not specific to Android. The Internal Console.WriteLine can be used to debug CoreLib on any platform.

Reply:
On Android, the canonical way to view logs seems to be through the Android Debug Bridge. I'm not sure if `pal_log.c's` `fwrite` makes it to that, and we have `AndroidLogPrint` to route to adb. So this change allows the Internal Console.Write to reach the adb logcat.

It is true that Internal.Console.Write/WriteLine is not specific to Android, but I hadn't yet found documentation explicitly saying to use Internal.Console.Write and figured that this could help anyone looking through CoreLib on Android. Should I remove this altogether?

Reply:
I think it is fine to add a note about `Internal.Console` somewhere under `docs/workflow/debugging`. I just think that it should be a general note and not android specific.

Reply:
I made it a general doc with a small note on where to see the log for Android. Should it be not be located under `libraries`?

Reply:
LGTM. Thanks!

> Should it be not be located under libraries?

I do not have strong opinion. I think it is fine where you put it.